### PR TITLE
kubeadm: cleanup prompt messages

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
kubeadm: cleanup prompt messages

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/join.go [hunk 1]
```diff
@@ -426,7 +426,7 @@ func (j *joinData) TLSBootstrapCfg() (*clientcmdapi.Config, error) {
 	if j.tlsBootstrapCfg != nil {
 		return j.tlsBootstrapCfg, nil
 	}
-	klog.V(1).Infoln("[preflight] Discovering cluster-info")
+	klog.V(1).Infoln("[preflight] discovering cluster-info")
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/join.go]: we decided that all messages that are in the form of `[area] Doing something` to have the "Doing" part starting with upper case.

Comment (SataQiu) [reply cmd/kubeadm/app/cmd/join.go]: Thanks @neolit123 ! 
Do we need to capitalize the first letter for all messages?
Or just leave them as they are?

Comment (neolit123) [reply cmd/kubeadm/app/cmd/join.go]: possibly, yes.
at least those that start with `[area]` should be.

File: cmd/kubeadm/app/cmd/join.go [hunk 2]
```diff
@@ -327,7 +327,7 @@ func newJoinData(cmd *cobra.Command, args []string, opt *joinOptions, out io.Wri
 		opt.externalcfg.Discovery.BootstrapToken = nil
 	} else {
 		if len(opt.cfgPath) == 0 && len(args) > 1 {
-			klog.Warningf("[preflight] WARNING: More than one API server endpoint supplied on command line %v. Using the first one.", args)
+			klog.Warningf("[preflight] WARNING: More than one API server endpoint supplied on command line %v. Using the first one", args)
```
Comment (timothysc) [inline cmd/kubeadm/app/cmd/join.go]: `WARNING: More than one API server endpoint supplied on command line %v, using the first argument.`

File: cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go [hunk 3]
```diff
@@ -55,16 +55,16 @@ func runCleanupNode(c workflow.RunData) error {
 	certsDir := r.CertificatesDir()
 
 	// Try to stop the kubelet service
-	klog.V(1).Infoln("[reset] Getting init system")
+	klog.V(1).Infoln("[reset] getting init system")
 	initSystem, err := initsystem.GetInitSystem()
 	if err != nil {
-		klog.Warningln("[reset] The kubelet service could not be stopped by kubeadm. Unable to detect a supported init system!")
-		klog.Warningln("[reset] Please ensure kubelet is stopped manually")
+		klog.Warningln("[reset] WARNING: The kubelet service could not be stopped by kubeadm. Unable to detect a supported init system!")
```
Comment (timothysc) [inline cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go]: Why add the extra warning, there will already be a [WARNING] prefix on the message.

**Review by timothysc — CHANGES_REQUESTED** [review]
/hold 

The changes in this PR were not log'd in an issue or an agreed upon item of work.  I don't agree with many of the changes in this PR.  Prior to submitting PRs please log and issue and get agreement on the work item prior to PR.

**Comment by SataQiu** [comment]
Thanks @neolit123 @timothysc 
This PR doesn't make sense, so close it.
/close

**Pull request closed without merging**